serverless/plugin/step_functions.json

```diff
@@ -1123,94 +1130,154 @@
+      }
+    },
+    "Catch": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "ErrorEquals": {
+            "type": "array",
+            "items": {
+              "$ref": "#/Aws_StateMachine_Errors"
+            }
+          },
+          "Next": {
+            "type": "string"
+          }
+        },
+        "required": ["ErrorEquals", "Next"]
+      }
+    }
+  },
+  "Aws_StateMachine_States_Map_ItemProcessor": {
     "type": "object",
     "properties": {
-      "Type": {
-        "type": "string",
-        "enum": ["Map"]
-      },
-      "Next": {
-        "type": "string"
-      },
-      "End": {
-        "enum": [true]
-      },
-      "Comment": {
-        "type": "string"
-      },
-      "OutputPath": {
-        "type": ["string", "null"]
-      },
-      "InputPath": {
-        "type": ["string", "null"]
-      },
-      "ResultPath": {
-        "type": ["string", "null"]
-      },
-      "ItemsPath": {
-        "type": ["string", "null"]
-      },
-      "MaxConcurrency": {
-        "type": "number",
-        "minimum": 0
-      },
-      "Iterator": {
-        "$ref": "#/Aws_StateMachine_Definition"
-      },
-      "Parameters": {
-        "type": ["object", "array"],
-        "items": {
-          "type": "object"
-        }
-      },
-      "ResultSelector": {
-        "type": "object"
-      },
-      "Retry": {
-        "type": "array",
-        "items": {
-          "type": "object",
-          "properties": {
-            "ErrorEquals": {
-              "type": "array",
-              "items": {
-                "$ref": "#/Aws_StateMachine_Errors"
+      "$ref": "#/Aws_StateMachine_States_Map_Properties",
+      "ItemProcessor": {
+        "$ref": "#/Aws_StateMachine_Definition",
+        "properties": {
+          "ProcessorConfig": {
+            "type": ["object"],
+            "properties": {
+              "Mode": {
+                "type": "string",
+                "enum": ["DISTRIBUTED", "INLINE"]
+              },
+              "ExecutionType": {
+                "type": "string",
+                "enum": ["STANDARD", "EXPRESS"]
               }
-            },
-            "IntervalSeconds": {
-              "type": "number",
-              "minimum": 0
-            },
-            "MaxAttempts": {
-              "type": "number",
-              "minimum": 0
-            },
-            "BackoffRate": {
-              "type": "number",
-              "minimum": 0
             }
-          },
-          "required": ["ErrorEquals"]
+          }
         }
       },
-      "Catch": {
-        "type": "array",
-        "items": {
-          "type": "object",
-          "properties": {
-            "ErrorEquals": {
-              "type": "array",
-              "items": {
-                "$ref": "#/Aws_StateMachine_Errors"
-              }
-            },
-            "Next": {
-              "type": "string"
-            }
+      "ItemBatcher": {
```

Comment:
This is because `ItemReader` and `ItemBatcher` are valid when `ItemProcessor` is present

Reply:
I don't understand it.